robottelo/ui/locators/menu.py

```diff
@@ -4,6 +4,16 @@
 from selenium.webdriver.common.by import By
 from .model import LocatorDict
 
+# This is the old menu container xpath, was failing in upstream
+# kept here for reference
+# MENU_CONTAINER_XPATH = (
+#     "//div["contains(@style,'static') or contains(@style,'fixed')]"
+# )
+
+# Downstream has 2 elements with id=menu we should get last()
+# which is the active menu
+MENU_CONTAINER_XPATH = '(//ul[@id="menu"])[last()]'
```

Comment:
yes, you get it right, if tomorrow we have one more element with id=menu, we will get completely failed run, exact addressing is always more reliable, but I think it is fine for current situation